partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java

```diff
@@ -356,7 +357,7 @@ public ResponseWrapper<PartnerCertDownloadResponeDto> getPartnerCertificate(
 	@RequestMapping(value = "/{partnerId}/originalPartnerCertificate", method = RequestMethod.GET)
 	@Operation(summary = "Service to get original partner certificate", description = "Service to get original partner certificate")
 	public ResponseWrapper<OriginalCertDownloadResponseDto> getOriginalPartnerCertificate(
-			@ApiParam("To download original partner certificate.")  @PathVariable("partnerId") @NotNull String partnerId) throws JsonParseException, JsonMappingException, JsonProcessingException, IOException {
+			@ApiParam("To download original partner certificate.")  @PathVariable("partnerId") @NotNull String partnerId) throws JsonParseException, JsonMappingException, JsonProcessingException, IOException, CertificateException {
```

Comment:
Have u tested the CertificateException case?
What is the API response in this case?

Reply:
Yes, Getting 200 response from API

Reply:
Add proper error code and error mesage